go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/alecthomas/chroma/v2
+module github.com/cloudbridgeuy/chroma/v2
```

Comment:
Remove this.

Reply:
Yes. Sorry, I was trying to see how this would work on the `feeze` project.